spec/models/profile_spec.rb

```diff
@@ -10,4 +10,16 @@
     profile = Profile.new(user_id: nil)
     expect(profile).to_not be_valid
   end
+
+  describe ".visible_for_user" do
+    it "returns the matching users profile or visible profiles" do
+      current_user = create(:user)
```

Comment:
I have a preference for the explicit `FactoryGirl.create` syntax.
more explicit, less magic